repository/src/main/resources/alfresco/public-services-security-context.xml

```diff
@@ -575,6 +575,7 @@
                 org.alfresco.service.cmr.search.CategoryService.deleteCategory=ACL_ALLOW
                 org.alfresco.service.cmr.search.CategoryService.getTopCategories=ACL_ALLOW
                 org.alfresco.service.cmr.search.CategoryService.getRootCategoryNodeRef=ACL_ALLOW
+                org.alfresco.service.cmr.search.CategoryService.getCategoriesCount=ACL_ALLOW
```

Comment:
Most likely - not needed - see my comment on `CategoryService` interface.

Reply:
Fixed